controlplane/kubeadm/internal/workload_cluster_etcd.go

```diff
@@ -88,6 +88,31 @@ func (w *Workload) EtcdIsHealthy(ctx context.Context) (HealthCheckResult, error)
 			response[name] = errors.Wrap(err, "failed to list etcd members using etcd client")
 			continue
 		}
+
+		// Check if any member's node is missing from workload cluster
+		// If any, delete it with best effort
+		for _, m := range members {
+			nodeName := m.Name
```

Comment:
I almost wonder if it's worth to just use `m.Name` and save the extra variable here